examples/lifetime_prediction/.dagianrc/bundle_config.yml

```diff
@@ -20,6 +20,9 @@ structure:
       args:
         dividend: weight
         divisor: height
+  unittest:
```

Comment:
don't do this, think of a meaningful word, and why do you need to add a new dataset for this?